### PR TITLE
Use img tag for images on module bar

### LINKED ISSUE #16917: Avatar / logo flashes on page changes
I believe it might have something to do with the fact that it tries to load from the API and inject as a data string, and the navigation causes an `update` cycle in Vue, which causes it to reinstantiate the image. 

### Discussed in https://github.com/directus/directus/discussions/16908

<div type='discussions-op-text'>

<sup>Originally posted by **ernst86913** December 24, 2022</sup>
Hi - 
currently the logo and the avatar are reloaded on every route change... which creates a bit of a "messy" experience for the user as those components are "flickering". Would it be possible to keep those 2 components stay on the screen like all other navigation bar components (e.g. the Directus Standard logo works nicely).

Thanks,
Roman </div>

### PR DESCRIPTION
**Pull request by licitdev**
## Description

The usage of `<v-image>` which initialises with an empty pixel resulted in the flashing. 
Using `` resolves this issue in a simple manner which utilises the browser cache.
Access token is to be added to the source for the avatar image only.

Note: This change removes the requests for the logo and avatar from the API request queue. 

Fixes #16917, fixes ENG-345.

## Type of Change

- [x] Bugfix
- [ ] Improvement
- [ ] New Feature
- [ ] Refactor / codestyle updates
- [ ] Other, please describe:

## Requirements Checklist

- [x] New / updated tests are included
- [x] All tests are passing locally
- [x] Performed a self-review of the submitted code

If adding a new feature:

- [ ] Documentation was added/updated. PR:

### CODE REVIEW

**Review by rijkvanzanten — DISMISSED** [review]
@licitdev Caching images can eat up a lot of memory really quickly. We need to have some sort of mechanism here to clear the cache once it hits a certain size, otherwise this'll easily turn into a memory leak

**Comment by licitdev** [comment]
Adding additional background info... 

This solution is done in this manner as the `private-view` is reloaded when the module is switched. That led to the reinitialisation of the `module-bar` and the nested `v-image` within. I think it might be due to how we're loading the modules, each containing a `private-view`, which resulted in the reload. 🤔 

Also, the `v-image` initializes with an `emptyPixel`, resulting in the flashing effect on load. The caching in this PR allows the `v-image` for module bar logo and avatar to display the cached image if it exists, skipping the `emptyPixel`.

**Comment by vanling** [comment]
This approach is nice =). Could a skipEmptyPixel prop not help here for the module-bar logo+avatar?

**Comment by rijkvanzanten** [comment]
I'm also very curious if we can figure out _why_ the v-image needs to re-render in the first place 🙂  I know the top-level private view is on a per-route, but if we can resolve this problem by not having it rerender things that don't need rerendering, we might be able to sneakily improve app performance too by having it do _less_ on each page navigation

**Comment by vanling** [comment]
Good question. it completely rerenders the `<private-view>` on every module change.

Are one of these props "new"?
https://github.com/directus/directus/blob/bb176c44fb06c7f95da6678f319d4575ddaea97b/app/src/views/private/private-view.vue#L10

I suspect the computed 'theme'

**Review by licitdev — COMMENTED** [review]
Refactored the PR to keep it simple.

File: app/src/views/private/components/module-bar-logo.vue [hunk 1]
```diff
@@ -12,7 +12,7 @@
 			<transition name="fade">
 				<v-progress-linear v-if="showLoader" indeterminate rounded @animationiteration="stopSpinnerIfQueueIsEmpty" />
 			</transition>
-			<v-image class="custom-logo" :src="customLogoPath" alt="Project Logo" />
+			<img class="custom-logo" :src="customLogoPath" alt="Project Logo" />
```
Comment (Nitwel) [inline app/src/views/private/components/module-bar-logo.vue:15]: I think you forgot to add the `addTokenToURL` function to the `customLogoPath` computed.

**Review by Nitwel — CHANGES_REQUESTED** [review]

File: app/src/views/private/components/module-bar-logo.vue (hunk 1 above)
Comment (licitdev) [reply app/src/views/private/components/module-bar-logo.vue:15]: The `project_logo` is publicly accessible, thus there's no need for the token to be added.

https://github.com/directus/directus/blob/89142d82bce614fc82a0168336c502aae8e365b9/api/src/services/assets.ts#L46-L51

https://github.com/directus/directus/blob/89142d82bce614fc82a0168336c502aae8e365b9/api/src/services/assets.ts#L62-L64

**Review by Nitwel — APPROVED** [review]
LGTM!

**Review by paescuj — APPROVED** [review]
Nice, looks good to me 👍

**Pull request merged**